mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java

```diff
@@ -161,15 +161,15 @@ public void onClick(DialogInterface dialog, int which) {
                     return;
                 }
                 mAmount = amount;
-                tvQrData.setText(qrData + ", " + mAmount);
+                tvQrData.setText(getString(R.string.vpa) + ": " + qrData + ", " + mAmount);
```

Comment:
The mentors have stated that we dont need the credential there, as it would disturb the process of scanning code.

Reply:
It is the same place, after the set amount dialog the QR details are set again that is why we need to set label again too otherwise the whole thing looks inconsistent. The scanning of the code takes place before this and does not utilize the `tvQrData`'s text it only uses the `qrData` which I am not modifying.

Reply:
Cool, if it doesn't disrupt :)